composer.json

```diff
@@ -3,7 +3,8 @@
     "description": "Asynchronous parallel Mysql client built on the Amp concurrency framework",
     "require": {
         "amphp/amp": "^2",
-        "amphp/socket": "^0.10"
+        "amphp/socket": "^0.10",
+        "phpbench/phpbench": "^0.13.0"
```

Comment:
This should go into `require-dev`, actually it's there, too, so delete it here.

Reply:
Fixed. Probably during merge I have added this in both sections.